nipype/interfaces/mrtrix3/reconst.py

```diff
@@ -39,8 +40,9 @@ class FitTensorInputSpec(MRTrix3BaseInputSpec):
         argstr='-method %s',
         desc=('select method used to perform the fitting'))
     reg_term = traits.Float(
-        5.e3, usedefault=True,
+        5.e3,
```

Comment:
Could you go ahead and remove the `5.e3`? The reason `usedefault=True` was added was because it looked like a default was intended by the original interface author. If we're not going to apply defaults, we should remove it.